recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
+{% set sha256 = "a14c20c61c086414895142d22d4ba9b4a5267a743dddfff860d947a5126fdb79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-fixed.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-fixed.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python 2.7*
```

Comment:
It looks like the newest version of `plumpy` supports python 3 (not clear from the pypi meta, but the `setup.py` mentions it), but perhaps this version does not. If this is a py27 package only, instead of using:

```
- python 2.7*
```

in the `build` section use the `skip` entry:

```
build:
   ...
  skip: True  # [not py27]
```